src/vision.cpp

```diff
@@ -549,6 +563,7 @@ void Vision::run()
       }
     }
     rclcpp::spin_some(node_);
+    max_pub_rate_->sleep();
```

Comment:
Is anything being caught in the sink or is it always pulling the latest image? ie, if you set the rate to something very very slow is it up to date our do we see lag?

Reply:
Looks like it pulls the latest image. I set the rate to 0.5 and I still see recent data